MyCompany.Seeq.Link.Connector.MyConnector/MyCompany.Seeq.Link.Connector.MyConnector.csproj

```diff
@@ -39,7 +39,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Seeq.Link.SDK" Version="66.0.0" />
+    <PackageReference Include="Seeq.Link.SDK" Version="100.0.0" />
```

Comment:
@lcollins-seeq , did the build succeed with just this change?

I expected you would need to:
- Use an actually released `100.0.x.x` Link SDK version, which in this case is [100.0.20047.37110](https://www.nuget.org/packages/Seeq.Link.SDK/100.0.20047.37110)
- Update packages.config to download that released version
- Do the same changes for the `MyCompany.Seeq.Link.Connector.MyConnector.Test` project